ts/sdk/src/interface.rs

```diff
@@ -57,3 +57,39 @@ pub struct Account {
     pub data: ByteBuf,
     pub owner: B58PK,
 }
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi, large_number_types_as_bigints)]
+#[serde(rename_all = "camelCase")]
+pub struct PoolState {
+    pub total_sol_value: u64,
+    pub trading_protocol_fee_bps: u16,
+    pub lp_protocol_fee_bps: u16,
+    pub version: u8,
+    pub is_disabled: u8,
+    pub is_rebalancing: u8,
+    pub admin: B58PK,
+    pub rebalance_authority: B58PK,
+    pub protocol_fee_beneficiary: B58PK,
+    pub pricing_program: B58PK,
+    pub lp_token_mint: B58PK,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi, large_number_types_as_bigints)]
+#[serde(rename_all = "camelCase")]
+pub struct LstState {
+    pub is_input_disabled: u8,
+    pub pool_reserves_bump: u8,
+    pub protocol_fee_accumulator_bump: u8,
+    pub sol_value: u64,
+    pub mint: B58PK,
+    pub sol_value_calculator: B58PK,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Tsify)]
+#[tsify(into_wasm_abi, from_wasm_abi, large_number_types_as_bigints)]
+#[serde(rename_all = "camelCase")]
+pub struct LstStateList {
+    pub states: Vec<LstState>,
+}
```

Comment:
are we able to return `Vec<LstState>` directly? Otherwise `LstStateList(Vec<LstState>)`?